hedera-node/src/main/java/com/hedera/services/state/merkle/internals/CopyOnWriteIds.java

```diff
@@ -172,9 +172,9 @@ public void setNativeIds(long[] ids) {
 		return ids;
 	}
 
-	public List<TokenID> getAsIds() {
+	public List<TokenID> getAsIds(int count) {
```

Comment:
Would it make sense to have two versions of this function?
`public List<TokenID> getAsIds()` to fetch all ids

and:
`public List<TokenID> getAsIds(int count)` 
?

Reply:
Actually we no longer need this. Removing it